internal/pkg/service/appsproxy/proxy/proxy_test.go

```diff
@@ -868,12 +844,12 @@ func TestAppProxyRouter(t *testing.T) {
 				require.NoError(t, err)
 				require.Equal(t, http.StatusInternalServerError, response.StatusCode)
 
-				// Request to private app
+				// Request to private app - unauthorized - selection page
 				request, err = http.NewRequestWithContext(context.Background(), http.MethodGet, "https://multi.hub.keboola.local/", nil)
 				require.NoError(t, err)
 				response, err = client.Do(request)
 				require.NoError(t, err)
-				require.Equal(t, http.StatusFound, response.StatusCode)
+				require.Equal(t, http.StatusUnauthorized, response.StatusCode)
```

Comment:
No redirect to the selector page, but it is rendered directly (more occurences).